lib/db/jdb-entry.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * JULEA - Flexible storage framework
+ * Copyright (C) 2019 Benjamin Warnke
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * \file
+ **/
+
+#include <julea-config.h>
+
+#include <glib.h>
+
+#include <string.h>
+
+#include <bson.h>
+
+#include <julea.h>
+#include <db/jdb-internal.h>
+#include <julea-db.h>
+#include "../../backend/db/jbson.c"
```

Comment:
We should find a better place for the wrapper. This can wait until after the PR is merged, though.